### PR TITLE
Update admin-mobile-app.md

### PR DESCRIPTION
**Pull request by msbemba**
Added information about changing language in the admin app

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3575

File: microsoft-365/admin/admin-overview/admin-mobile-app.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ Yes, but with reduced functionality. You'll be able to sign in and view service
   
 ### What languages are supported by the app?
 
-The app supports all 39 languages that the web-based Microsoft 365 admin center supports. 
+The app supports all 39 languages that the web-based Microsoft 365 admin center supports. To change the language, select the app launch icon in the upper bar and click on settings and Language.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/admin-overview/admin-mobile-app.md]:
```suggestion
The app supports all 39 languages that the web-based Microsoft 365 admin center supports. To change the language, select the app launch icon in the upper bar and choose **Settings** > **Language**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**